debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -680,6 +749,17 @@ public static OversizeHandlingMode parse(String value, String defaultValue) {
             .withInvisibleRecommender()
             .withDescription("A comma-separated list of regular expressions that match the collection names for which changes are to be excluded");
 
+    public static final Field FILTERS_MATCH_MODE = Field.create("filters.match.mode")
+            .withDisplayName("Database and collection include/exclude match mode")
+            .withEnum(FiltersMatchMode.class, FiltersMatchMode.REGEX)
+            .withGroup(Field.createGroupEntry(Field.Group.FILTERS, 6))
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.MEDIUM)
+            .withDescription("The mode used by the aggregation pipeline to match events based on  included/excluded database and collection names"
+                    + "Options include: "
+                    + "'regex' (the default) Database and collection includes/excludes are evaluated as regular expressions; "
+                    + "'literal' Database and collection includes/excludes are evaluated as simple string literals; ");
```

Comment:
Can we add a comment about whitespace and comma separated values?

Reply:
Not sure I follow

Reply:
Well, it's not 100% literal. There is a format that is used for parsing. We should mention this and call out if it is whitespace sensitive or not.